### ISSUE TITLE
MKVmerge not found[BUG] 

### ISSUE THREAD

**Issue by RoseB11** [anidl/funimation-downloader-nx#79]
** Main info: **

Script version: 4.8.0
Show ID: 1400603
Episode ID: 1-6

** Additional Info: **
Hello,
This is my first time using this downloader so I may have missed something along the way. Whatever I try to download, I always get an mkv file and a subtitle file separately. After these files download, I get a warning that MKVmerge is not found even though I have mkvmerge.exe, and ffmpeg.exe in the bin folder for the downloader.  I am hoping to get 1 mp4 file with the subtitles included. Attached are screenshots of the folder and the error. Thanks

![Screenshot 2021-05-20 104256](https://user-images.githubusercontent.com/21056269/118999323-2beac980-b958-11eb-948f-11278cb365d3.png)

![image](https://user-images.githubusercontent.com/21056269/118998407-6dc74000-b957-11eb-95cf-0a060f70ba3b.png)

**Comment by izu-co** [comment]
Alright, first it seems that you are using my fork. Just as a quick question what exactly are you using (the release or the code base).

As for the mkvmerge, it seems that you have to put it into ```/bin/mkvtoolnix/mkvmerge.exe``` or just change the mkvmerge path in the ```bon-path.yml``` file to ```./bin/mkvmerge.exe```.

As for why the subtitles aren't included, try adding ```--mks``` (here the description ```Add subtitles to mkv/mp4 (if available)```)

**Comment by RoseB11** [comment]
I'm not fully sure where to find the release version. I downloaded everything today so it should be the latest.

After relocating the .exe file and adding --mks, I no longer get the mkvmerge not found error but now another error pops up. The subtitle file cannot be opened for reading: open file error. I have attached the error for your reference.


 
![image](https://user-images.githubusercontent.com/21056269/119014185-b554c880-b965-11eb-94ad-3abf33b7f532.png)
